krun.py

```diff
@@ -92,20 +113,24 @@ def run(self):
 
         # Rough ETA execution timer
         exec_start_rough = time.time()
-        stdout = vm_def.run_exec(entry_point, self.benchmark, self.vm_info["n_iterations"],
-                                 self.parameter, heap_limit_kb)
+        stdout, stderr = vm_def.run_exec(entry_point, self.benchmark, self.vm_info["n_iterations"],
+                                         self.parameter, heap_limit_kb)
         exec_time_rough = time.time() - exec_start_rough
 
         try:
             iterations_results = eval(stdout) # we should get a list of floats
         except SyntaxError:
-            error("=ERROR=" * 8)
-            error("*error: benchmark didn't error a parsable list.")
-            error("We got:\n---\n%s\n---\n" % stdout)
-            error("To see the invokation set the BENCH_DEBUG env and run again")
-            error("=ERROR=" * 8)
-
+            rule = 50 * "-"
+            err_s = "Benchmark didn't emit a parsable list on stdout.\n"
```

Comment:
I did start out with a triple quoted string, but to achieve the desired effect I would have to mess up the indentation of the file (the triple quoted text would have to be up against the left margin). What do you think?